docs/user/reference/metrics/event.md

```diff
@@ -187,6 +190,10 @@ var snapshot = views::login_opened.test_get_value(None).unwrap();
 assert_eq!(2, snapshot.len());
 let first = &snapshot[0];
 assert_eq!("login_opened", first.name);
+
+let extra = event.extra.unwrap();
+assert_eq!(1, extra.len());
```

Comment:
We put a `len()` function on the event extra impl?

Reply:
So I cargo culted this from the Rust test for the event metric, as it happens:

https://github.com/mozilla/glean/blob/main/glean-core/tests/event.rs#L80

this both checks the length and then grabs the extra ala the syntax I used in the next line. I'm happy to change this though, if we don't like that.

Reply:
Updated this to reflect your suggestion. I don't think the length assertion is at all useful and if there is a direct access I think we should rec that as it will potentially work with folks' autocomplete and maybe the compiler will be more useful in that case.

Reply:
Oh, huh, the `test_get_value` on the trait returns a Vec<RecordedEvent> instead of a Vec<T>. Weird.